packages/preview/tufte-memo/0.1.0/template/main.typ

```diff
@@ -1,4 +1,4 @@
-#import "@preview/tufte-memo:0.1.0": *
+#import "@preview/tufte-memo:0.1.2": *
```

Comment:
This file should not change, this version has been published and can't be changed anymore.